DataFormats/MemoryResources/include/MemoryResources/MemoryResources.h

```diff
@@ -39,6 +39,7 @@
 #include <unordered_map>
 #include "FairMQMessage.h"
 #include "FairMQTransportFactory.h"
+#include "MemoryResourceTools.h"
```

Comment:
```suggestion
#include <fairmq/MemoryResourceTools.h>
```

Convention: For old-style headers that include the prefix `FairMQ` in the filename, don't use `fairmq/` directory, for new-style headers without `FairMQ` prefix in their filename, use `fairmq/` directory prefix.

And besides, why do people use `#include "..."` for external headers (in HEP software), it will always fall back to the meaning of `#include <...>` anyways, why not write it in the first place? We also use it for boost or stl headers.

Reply:
https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=Names_and_Order_of_Includes#Names_and_Order_of_Includes

Reply:
@dennisklein old habits die hard... will fix.